src/lighteval/models/transformers/delta_model.py

```diff
@@ -87,3 +88,11 @@ def _create_auto_model(
         )
 
         return model
+
```

Comment:
Same, I could open another PR for this, it was bloating the pipeline code but it's unrelated to the PR